ddtrace/contrib/internal/aiobotocore/patch.py

```diff
@@ -45,6 +46,11 @@
     },
 )
 
+def _instrumented_versions() -> Dict[str, Collection[str]]:
```

Comment:
```suggestion
def _supported_versions() -> Dict[str, Collection[str]]:
```

? "instrumented" seems like an action has been taken, "these versions have been instrumented" vs "these are the versions we are able to instrument"

Reply:
fair